tests/test_anonymize.py

```diff
@@ -61,6 +79,25 @@ def test_anonymize(test_dicom) -> None:
         assert block[i + 1].value == element.value
 
 
+def test_anonymize(test_dicom) -> None:
+    # Additional testing is present in the `dicom-anonymizer` repo
+    # This test is more of a sanity check
+    ds = copy.deepcopy(test_dicom)
+
+    for tag in CRITICAL_PHI_TAGS:
+        s = "19000101" if "date" in repr(tag).lower() else "filler string"
```

Comment:
Note that without a date shifting strategy we will not be able to associate dated reports with studies using date alone, nor will we be able to track the chronology of studies for a given patient

Reply:
Let's postpone implementing a date shifting strategy since it introduces risk to exposing PHI.